docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1142,6 +1142,18 @@ The following error message codes may appear on the app-server log. Use the foll
 
   No action Required.
 
+### ZWED0303I
+
+  Plugin _plug-in_ will not be loaded because no dataservices requested it
+
+  **Reason:**
+
+  Zowe displays this message in the Zowe log file after Zowe detects that no dataservices currently deployed within Zowe requested `org.zowe.zlux.auth.trivial` or any other _plugin_ to authenticate users.
```

Comment:
This is about SECURITY plugins specifically.
Security plugins only load when something needs them.

> `org.zowe.zlux.auth.trivial` or any other _plugin_
Technically true, but trivial is not special, it's just like any other security _plugin_.